providers/azuredns/azureDnsProvider.go

```diff
@@ -73,6 +73,10 @@ func init() {
 }
 
 func (a *azureDNSProvider) getExistingZones() (*adns.ZoneListResult, error) {
+    // Please note — this function doesn't work with > 100 zones
```

Comment:
Can we just delete `getExistingZones()`?  (considering that it has a known but and it isn't used)

Reply:
I would like to test a new (upcoming) Azure SDK version. It can probably fix a `Complete` calls. That's the only one reason to keep this function at least for some time.
But we can restore it at any moment and for clarity reason better to delete it.

Reply:
Sounds good. Let me know when it is ready for merge.

Reply:
It's ready (and tested on Azure) right now. And I'll test new Azure SDK in few days/weeks when it will be released.